tools/data/kinetics/generate_rawframes_filelist.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+DATASET=$1
```

Comment:
whether we should guard against bad arguments, something like (not tested)
 
```bash
if [ $# -eq 0 ]; then
    echo "No arguments provided"
    exit 1
fi
```

Reply:
I plan to use something like the following script to test the validity of the argument. [Maybe ugly]

```
DATASET=$1
if [ "$DATASET" == "kinetics400" ] || [ "$1" == "kinetics600" ] || [ "$1" == "kinetics700" ]; then
        echo "We are processing $DATASET"
else
        echo "Bad argument"
        exit 0
fi
```